source/AndroidXProject.cshtml

```diff
@@ -94,9 +94,9 @@
 
   <ItemGroup>
     <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\monoandroid90" />
-    <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\net6.0-android30.0" />
+    <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\net6.0-android31.0" />
```

Comment:
You might be able to do:
```suggestion
    <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\net6.0-android$(TargetPlatformVersion)" />
```

Reply:
That would be nicer, lemme kick off a build with that and see if it works.